tollway/events.py

```diff
@@ -38,11 +48,14 @@ def create_duplicate_event(events_log: EventsLog) -> dict[str, str]:
     return duplicate_event
 
 
-def process_duplicate_event(events_log: EventsLog, publisher: PublisherClient, topic_path: Topic) -> EventsLog:
+def process_duplicate_event(
+    events_log: EventsLog, publisher: PublisherClient, topic_path: Topic
+) -> EventsLog:
     duplicate_event = create_duplicate_event(events_log)
     if publisher and topic_path:
         data = encode_message(message=duplicate_event)
         future = publisher.publish(topic=topic_path, messages=data)
+    duplicate_event["is_duplicate"] = "true"
```

Comment:
this is json true?

Reply:
I need to rework this. I originally had as `True` but received an error from mypy so changed to string. I no longer received the error but as I'm looking into it there's a lot more going on with typing that needs to be tinkered with.

Reply:
maybe the return of `create_duplicate_event` was typed as a dict of strings then?

Reply:
yeah and yesterday I updated the typing and added `Union[str, bool]` in a few different spots and it caused a typing explosion. I'm working through it along with refactoring `events` module into classes.